src/database.ts

```diff
@@ -821,7 +822,14 @@ class Database extends common.GrpcServiceObject {
         callback!(err as ServiceError, null, undefined);
         return;
       }
+
+      const span = getActiveOrNoopSpan();
+      span.addEvent('Creating BatchTransaction');
```

Comment:
Remove this event as I feel once we add the spans , there should be a span for "BatchTransaction" , so this information will be repeated in this case.

Reply:
Alright, thanks! Though in our scope document we needed to annotate when a transaction is started. Correct though that we shall have the span for BatchTransaction.